packages/logging/src/log.js

```diff
@@ -589,12 +589,27 @@ Log.prototype.write = function(entry, options, callback) {
     options = {};
   }
 
-  this.decorateEntries_(arrify(entry), function(err, decoratedEntries) {
-    // Ignore errors (the API will speak up if it has an issue).
+  if (!options.resource) {
```

Comment:
Sorry, I see the difference now, of course. Since `assignDefaultResource` is no longer used, would you mind removing it?